_includes/cookie-notice.html

```diff
@@ -57,23 +62,38 @@
     border: 1px solid var(--post-link-hover);
   }
 
+  @media (min-width: 1280px) {
+    #cookie-notice>div {
+      padding-right: 6vw;
+    }
+  }
   @media (max-width: 767px) {
-    #cookie-notice {
-      gap: 20px;
+
+    #cookie-notice>div {
       flex-wrap: wrap;
+      flex-direction: column;    
+      padding-right: 0;
+      gap: 20px;
     }
 
     #cookie-notice p span {
       display: none;
     }
   }
 </style>
-
 <div id="cookie-notice">
-  <p>This website uses cookies to ensure you get the best browsing experience. By continuing to use this website, you consent to our use of these cookies. <span>This website contains proprietary content that belongs to LivePerson and that is intended for educational use by our customers and prospects. Your use of this site is subject to our <a href="https://www.liveperson.com/policies/apitou/" target="_blank">Terms of Use</a>, which include restrictions on any use of our information for unauthorized purposes.</span></p>
-  <a id="cookie-notice-accept" class="btn">I agree</a>
+    <div class="flex max-width">
```

Comment:
Were the changes here intentional, or due to tooling?

This will lead to issues (note the comma on a new line)—please revert, or suggest a new indentation style.

Reply:
Thanks for fixing the imminent comma-whitespace issue. Now, what logic do the other changes follow?

I suggest to revert the formatting changes for this text block and to propose a new standard for how to format text first. It would simplify handling this PR and allow us to look into what you like to accomplish.